src/convertToObject.js

```diff
@@ -3,16 +3,31 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see
+ * an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  const splittedString = sourceString.split(';');
+
+  return splittedString.reduce((acc, word) => {
+    const [key, value] = word
+      .split(':')
+      .map(item => item.trim());
+
+    if (key && value) {
+      acc[key] = value;
+    }
```

Comment:
I think it would be better to avoid some nested iterations in reduce. 

```suggestion
    const [key, value] = word.split(':')

    if (key && value) {
      acc[key.trim()] = value.trim();
    }
```